tests/integration/features/command/command-list.feature

```diff
@@ -0,0 +1,21 @@
+Feature: command/command-list
+
+  Scenario: Get the output options
+    Given invoking occ with "talk:command:list --help"
+    Then the command was successful
+    And the command output contains the text "Output format"
+
+  Scenario: List all available commands as plain text (default)
+    Given invoking occ with "talk:command:list"
+    Then the command was successful
+    And the command output contains the text "Response values: 0 - No one"
+
+  Scenario: List all available commands as plain text
+    Given invoking occ with "talk:command:list --output=plain"
+    Then the command was successful
+    And the command output contains the text "Response values: 0 - No one"
+
+  Scenario: List all available commands as json
+    Given invoking occ with "talk:command:list --output=json"
+    Then the command was successful
+    And the command output contains the text "[{"
```

Comment:
Will be a bit complex to match a string with quote because the regex ignore quotes: `([^"]*)`. If we change the regex don't will be possible found the end of string and maybe we will need to change the delimiter
of the json string:

https://stackoverflow.com/questions/8533943/cucumber-fill-in-a-field-with-double-quote-in-it#8534220

I changed to accept the argument also as `PyStringNode` and don't need do nothin more inside the method because the `PyStringNode` class have a magich methdo `__toString()`.

The problem that occur when I tried to escape the double quote:
![Screenshot_20230426_071759](https://user-images.githubusercontent.com/1079143/234546106-c7aa2802-5eba-42a1-b81e-1345a86a994f.png)

The solution:

```gherkin
And the command output contains the text
  """
  "name":"talk","command":"help","script":"help"
  """
```
https://docs.behat.org/en/v2.5/guides/1.gherkin.html#pystrings